debian/changelog

```diff
@@ -1,3 +1,10 @@
+pop-gnome-shell-theme (4.0.10) cosmic; urgency=medium
+
+  * Fix invisible switches in dark variant (#66)
+  * Ensure Search text color is correct (#63)
+
```

Comment:
Not adding the build-dep change here because that doesn't affect the Debian Packaging.